src/lib/components/support.svelte

```diff
@@ -1,123 +1,159 @@
-            target="_blank"
-            rel="noopener noreferrer"
-            class="button is-secondary u-padding-inline-12 u-stretch u-main-center u-gap-4 u-flex-basis-auto">
-            <span class="icon-github" aria-hidden="true" />
-            <span class="text">Open issue</span>
-        </a>
-    </div>
-</section>
-<section class="drop-section u-grid u-gap-8 u-padding-24">
-    <div>
-        <h4 class="eyebrow-heading-3">Community support</h4>
-        <p class="text u-margin-block-start-8">Get help from our community</p>
+                    <div class="u-flex u-gap-4 u-cross-center">
+                        {#if isSupportOnline()}
+                            <span
+                                class="icon-check-circle u-color-text-success"
+                                aria-hidden="true" />
+                        {/if}
+
+                        {supportTimings}
+                    </div>
+                </div>
+            {:else}
+                <Button
+                    href={option.link}
+                    external
+                    secondary
+                    class="u-flex u-cross-center u-gap-6"
+                    on:click={() => {
+                        trackEvent('click_organization_upgrade', {
+                            from: 'button',
+                            source: 'support_menu'
+                        });
+                    }}>
+                    <span class={`icon-${option.icon}`} />
+                    <span>{option.cta}</span>
+                </Button>
+            {/if}
+        </Card>
+    {/each}
+
+    <div class="u-width-full-line">
+        {#key $app.themeInUse}
+            <iframe
+                style="color-scheme: none"
+                title="Appwrite Status"
+                src={`https://status.appwrite.online/badge?theme=${
+                    $app.themeInUse === 'dark' ? 'dark' : 'light'
+                }`}
+                width="250"
+                height="30"
+                frameborder="0"
+                scrolling="no">
+            </iframe>
+        {/key}
     </div>
-    <ul class="u-flex u-gap-8">
-        <li>
-            <Button href="https://github.com/appwrite" text noMargin external ariaLabel="Github">
-                <span class="icon-github" aria-hidden="true" />
-            </Button>
-        </li>
-        <li>
-            <Button
-                href="https://appwrite.io/discord"
-                round
-                text
-                noMargin
-                external
-                ariaLabel="Discord">
-                <span class="icon-discord" aria-hidden="true" />
-            </Button>
-        </li>
-    </ul>
 </section>
+
+<style>
+    .support-section {
+        gap: 1rem;
+        padding: 20px;
+        display: flex;
+        flex-direction: column;
+    }
+
+    @media (max-width: 768px) {
+        .support-section {
+            gap: 1.25rem;
+            padding: 0.5rem;
+        }
+    }
+</style>
```

Comment:
Let's change this to lang='scss', and then use the nesting to not have to retype the class name.